changelog

```diff
@@ -1,3 +1,202 @@
+      * Game actions can now play music with a `music` node. (@eebop)
+      * Various new cloak related attributes. (@EjoThims, @Hurleveur, @petervdmeer, @quyykk, @TomGoodIdea, @warp-core, @Zitchas)
+      * Added the "disabled recovery time" attribute which allows a disabled ship to repair itself, with an optional energy and heat cost. (@Azure3141)
+      * Allow plugins to add more metadata to their plugin.txt file. (@Beed-git)
+      * Events can now be made to apply instantly with a delay of 0 days. (@RisingLeaf)
+      * Mission triggers and actions can now give the player debt. (@Amazinite)
+      * Missions can mark systems of importance. (@Amazinite)
+    * Mechanics changes:
+      * Remove the forced-default disabled hails from Government. (@Amazinite)
+      * NPC scan and surveillance personality behavior changes. (@Amazinite)
+      * Auto-pilot now stops thrusting upon reaching 99% of max speed to save energy. (@samrocketman)
+  * User interface:
+    * Include messages when hailing other ships in the message log. (@TomGoodIdea)
+    * The shop now has tooltips for viewing your unabbreviated credits and the names of your ships without needing to select them. (@Amazinite)
+    * Disabled hails are now sent passively. (@bene-dictator)
+    * Hide unknown system names in the escort HUD. (@TomGoodIdea)
+    * Allow multiple pages of controls. (@TomGoodIdea)
+    * A help message for managing multiple ships in the outfitter. (@TomGoodIdea)
+    * Added a preference to turn off the sobel filter on your flagship and target ship in the HUD. (@warp-core)
+    * Added a control for toggling the turret tracking setting. (@TomGoodIdea)
+    * "Fancy" cloaked ship outlines. (@Koranir)
+    * Expand the orbits key image to make space for all vanilla governments. (@lantzk)
+    * Added a camera acceleration setting that causes the camera to shift with the flagship's movement. (@Koranir, @tibetiroka)
+  * Under the hood:
+    * Improve load panel tooltip timestamp generation. (@RisingLeaf, @ziproot)
+    * Style updates to Armament. (@RisingLeaf)
+    * General purpose TaskQueue instead of SpriteQueue. (@quyykk)
+    * Fix typo in and add Hungarian language description to Unix metadata file. (@ovari)
+    * Don't recalculate an escort's distance map every frame if its target system already matches the flagship. (@Koranir)
+  * CI/CD and development environment:
+    * Stricter copyright checking. (@tibetiroka)
+    * Don't check for leading/trailing spaces in substitutions defined in missions. (@tibetiroka)
+    * Support CodeBlocks no longer stand-alone, but only through CMake. (@quyykk)
+    * Check copyright with a linter. (@tibetiroka)
+    * Link GLEW::glew instead of GLEW::GLEW in CMakeLists.txt. (@Koranir)
+    * Simplify CMake build on Linux and MacOS. (@quyykk)
+    * Run CI workflows when their configuration file is changed. (@tibetiroka)
+    * Update codespell check exclusion list. (@tibetiroka)
+
```

Comment:
```suggestion
  * Game mechanics:
    * New mechanics:
      * Autocondition for testing if you can land on a given planet. (@Amazinite)
      * Added a "fused" weapon tag that causes projectiles to explode at the end of their life. (@Amazinite)
      * Added tags that stop projectiles from colliding with different object types. (@Amazinite)
      * Added a "quiet" personality that prevents passive hails but still allows direct hails. (@bene-dictator)
      * Added support for substituting the values of conditions in word form within conversation text. (@UnorderedSigh)
      * Separated delayed and non-delayed shield generation and hull repair attributes. (@Quantumshark)
      * "image" interface elements can be rotated. (@warp-core)
      * Boarding (repair or capture) ships with a fighter flagship is now possible. (@tibetiroka)
      * Game actions can now play music with a `music` node. (@eebop)
      * Various new cloak related attributes. (@EjoThims, @Hurleveur, @petervdmeer, @quyykk, @TomGoodIdea, @warp-core, @Zitchas)
      * Added the "disabled recovery time" attribute which allows a disabled ship to repair itself, with an optional energy and heat cost. (@Azure3141)
      * Plugins can now add more metadata to their plugin.txt file. (@Beed-git)
      * Events can now be made to apply instantly with a delay of 0 days. (@RisingLeaf)
      * Mission triggers and actions can now give the player debt. (@Amazinite)
      * Missions can now mark systems of importance. (@Amazinite)
    * Mechanics changes:
      * Removed the forced-default disabled hails from Government. (@Amazinite)
      * Various NPC scan and surveillance personality behavior changes. (@Amazinite)
      * Auto-pilot now stops thrusting upon reaching 99% of max speed to save energy. (@samrocketman)
  * User interface:
    * Include messages when hailing other ships in the message log. (@TomGoodIdea)
    * The shop now has tooltips for viewing your unabbreviated credits and the names of your ships without needing to select them. (@Amazinite)
    * Disabled hails are now sent passively. (@bene-dictator)
    * Hid unknown system names in the escort HUD. (@TomGoodIdea)
    * There can now be multiple pages of controls. (@TomGoodIdea)
    * There is now a help message for managing multiple ships in the outfitter. (@TomGoodIdea)
    * Added a preference to turn off the sobel filter on your flagship and target ship in the HUD. (@warp-core)
    * Added a control for toggling the turret tracking setting. (@TomGoodIdea)
    * "Fancy" cloaked ship outlines. (@Koranir)
    * Expanded the orbits key image to make space for all vanilla governments. (@lantzk)
    * Added a camera acceleration setting that causes the camera to shift with the flagship's movement. (@Koranir, @tibetiroka)
  * Under the hood:
    * Improved load panel tooltip timestamp generation. (@RisingLeaf, @ziproot)
    * Some style updates to Armament. (@RisingLeaf)
    * Use general purpose TaskQueue instead of SpriteQueue. (@quyykk)
    * Fixed typo in and add Hungarian language description to Unix metadata file. (@ovari)
    * Don't recalculate an escort's distance map every frame if its target system already matches the flagship. (@Koranir)
  * CI/CD and development environment:
    * Stricter copyright checking. (@tibetiroka)
    * Don't check for leading/trailing spaces in substitutions defined in missions. (@tibetiroka)
    * Support CodeBlocks no longer stand-alone, but only through CMake. (@quyykk)
    * Check copyright with a linter. (@tibetiroka)
    * Link GLEW::glew instead of GLEW::GLEW in CMakeLists.txt. (@Koranir)
    * Simplified CMake build on Linux and MacOS. (@quyykk)
    * Run CI workflows when their configuration file is changed. (@tibetiroka)
    * Updated codespell check exclusion list. (@tibetiroka)

```